### PR TITLE
:sparkles: page user add

### LINKED ISSUE #319: admin: data management: personeel and syndici forms

### LINKED ISSUE #333: Service: User
Add following features to User service:
- Patch

### PR DESCRIPTION
**Pull request by milachae**
Closes #333, Closes #319 

This PR includes the following things:
- accept a user when he was registered himself.
- update region and function of a specific user.
- View additional data of the users.

### CODE REVIEW

File: frontend/src/components/LayoutDataToevoegen.jsx [hunk 1]
```diff
@@ -112,21 +113,33 @@ function buildingList(data) {
...
+            <div className={"flex flex-row items-center"}>
+              <p className={"flex-grow"}>
+                {data.first_name + " " + data.last_name}
+              </p>
+              <ColoredTag
+                className={`${
+                  data.active ? "text-good-1 bg-good-2" : "text-bad-1 bg-bad-2"
+                }`}
+              >
+                {data.active ? "Active" : "Inactive"}
```
Comment (bertcappelle) [inline frontend/src/components/LayoutDataToevoegen.jsx:135]: Translate to Dutch

**Review by renevds — CHANGES_REQUESTED** [review]
Can you make the text of disabled fields (like name and stuff) a lighter colour.

File: frontend/src/components/LayoutDataToevoegen.jsx (hunk 1 above)
Comment (milachae) [reply frontend/src/components/LayoutDataToevoegen.jsx]: This is fixed now!

**Comment by milachae** [comment]
@renevds 
> Can you make the text of disabled fields (like name and stuff) a lighter colour.

I already tried to make it a lighter colour, but the other possible options was to light.
image

I can add a new colour to fix this?

**Comment by renevds** [comment]
A new colour sounds good to me!

**Review by bertcappelle — APPROVED** [review]

**Comment by milachae** [comment]
@renevds 

I tried adding a new colour:
image

**Review by renevds — APPROVED** [review]
GJ

**Pull request merged**